app/controllers/coronavirus_pages_controller.rb

```diff
@@ -108,7 +108,7 @@ def valid_content?(content, type)
     return false if content.nil?
 
     required_keys =
-      type == :landing ? required_landing_page_keys : required_hub_page_keys
+      type.to_sym == :landing ? required_landing_page_keys : required_hub_page_keys
```

Comment:
we might be able to get rid of some of these checks if we can move towards defining a govuk-content-schemas schema

Reply:
Yes - that is the intention. Just not in this PR :)